js_pt1/dest/desktop/js/kintoneRESTAPI.js

```diff
@@ -0,0 +1,47 @@
+(function() {
+	'use strict';
+	kintone.events.on('app.record.create.show', function(event) {
+		const params = {
+			'app': event.appId
+		};
+		kintone.api(kintone.api.url('/k/v1/app/form/fields'), 'GET', params).then(function(resp) {
+			const setRecords = kintone.app.record.get();
+			const actions = [];
+			const options = resp.properties.table.fields.action_5.options;
+			for (let i = 0; i < 6; i++) {
```

Comment:
It will not work if the number of table rows is more than 6.
Please fix the code so that it works well regardless of the number of table rows.